src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -137,6 +139,45 @@ public static String extractVersionNumber(String inputString) {
         return inputString.substring(startPos, endPos).replace("-", ".").trim();
     }
     
-    
+    /**
+     * Verifies that the test properties specified in test.properties file allows for HTML
+     * regeneration via God mode to work smoothly (i.e all test HTML files are correctly regenerated, 
+     * strings that need to be replaced with placeholders are correctly replaced, and 
+     * strings that are not supposed to be replaced with placeholders are not replaced).
+     */
+    public void verifyReadyForGodMode() {
+        if (!inst().isDevServer()) {
+            Assumption.fail("God mode regeneration works only in dev server.");
+        }
+        if (!areTestAccountsReadyForGodMode()) {
+            Assumption.fail("Please append a unique id to each of the default account in test.properties "
+                            + "in order to use God mode, e.g change alice.tmms to alice.tmms.yourName, "
```

Comment:
I am still wondering if we can make it blatant and say "In order to use God mode, append your name to each of the default accounts in test.properties to make them unique. e.g ...". 
And make yourName within angular brackets <> to make it more obvious. What do you think?